pennylane/devices/default_qubit.py

```diff
@@ -502,7 +502,10 @@ def preprocess(
 
         transform_program.add_transform(validate_device_wires, self.wires, name=self.name)
         transform_program.add_transform(
-            mid_circuit_measurements, device=self, mcm_config=config.mcm_config
+            mid_circuit_measurements,
+            device=self,
+            mcm_config=config.mcm_config,
+            interface=config.interface,
```

Comment:
Note that the same change needs to be made to `lightning.qubit` and `lightning.kokkos` preprocessing. We should open a PR in lightning in parallel to avoid errors once this PR is merged.

Reply:
Has a PR been opened? Or is this tracked elsewhere?

Reply:
I believe there is a story on shortcut for it, but not a PR yet? @mudit2812

Reply:
There is a [story](https://app.shortcut.com/xanaduai/iteration/61839?group_by=owner_ids&ct_workflow=all) and [open PR](https://github.com/PennyLaneAI/pennylane-lightning/pull/760).